game_of_thrones/app/views/houses/edit.html.erb

```diff
@@ -0,0 +1,23 @@
+<h2>Update House</h2>
+
+<form action="/houses/<%= @house.id %>" method="post">
+  <input type="hidden" house_name="authenticity_token" value="<%= form_authenticity_token %>">
```

Comment:
You can also and should use `form_for`, but you can use the `form_authenticity_token` view-helper method.